strata/cmd/mongo/lreplica_ossstorage_driver/lrossdriver/lrossdriver.go

```diff
@@ -3,24 +3,24 @@
 //  LICENSE file in the root directory of this source tree. An additional grant
 //  of patent rights can be found in the PATENTS file in the same directory.
 
-package lrs3driver
+package lrossdriver
 
 import (
 	"errors"
 	"os"
 	"strconv"
 
-	"github.com/AdRoll/goamz/aws"
-	"github.com/AdRoll/goamz/s3"
+	"github.com/PinIdea/oss-aliyun-go"
 	"github.com/facebookgo/rocks-strata/strata"
+
 	"github.com/facebookgo/rocks-strata/strata/mongo/lreplica"
-	"github.com/facebookgo/rocks-strata/strata/s3storage"
+	"github.com/naytzyrhc/rocks-strata/strata/ossstorage"
 )
 
 // AWSOptions are common to all commands
-type AWSOptions struct {
-	Region       string `short:"R" long:"region" description:"AWS region name, such as \"us-east-1\"" default:"us-east-1"`
-	BucketName   string `short:"b" long:"bucket" description:"Name of S3 bucket used to store the backups" required:"true"`
+type OSSOptions struct {
```

Comment:
on the other hand actually looking on what the struct contains (region, bucketname, bucketprefix, bucketacl), it's more OSS options than general Aliyun options...